### PR TITLE
Update ujson dependency

### LINKED ISSUE #802: Adapt to ujson > 2.0
ujson 2.0 has been released, but this package requires <= 1.35 currently. Please update to the latest version.

### LINKED ISSUE #857: Pip install fails with ujson error
Default pip installation fails with error for wrong `ujson` version.

`pip install python-language-server`

**Error:**
```
python-jsonrpc-server 0.4.0 requires ujson>=3.0.0, but you'll have ujson 1.35 which is incompatible.
```

Since the dependency lib `python-jsonrpc-server` was updated to v0.4 the updated ujson version leads to an incompatible installation with current settings ujson 1.35 in pyls.

Latest version of python-jsonrpc-server: https://pypi.org/project/python-jsonrpc-server/
Depending pr: https://github.com/palantir/python-jsonrpc-server/pull/44/files

**Workaround:** use `python_jsonrpc_server<0.4`

```
pip install python-jsonrpc-server<0.4
pip install python-language-server
```

**Possible solutions:**
* pin python_jsonrpc_server to v <0.4
* update `ujson` too

Maybe blocked by https://github.com/palantir/python-language-server/pull/808

Quick and easy fix by https://github.com/palantir/python-language-server/pull/859

### PR DESCRIPTION
**Pull request by steff456**
This PR updated the ujson dependency.

Fixes #857 
Fixes #802

### CODE REVIEW

**Review by delta003 — DISMISSED** [review]

**Comment by ccordoba12** [comment]
@delta003, a new release of python-jsonrpc-server is necessary for this one to work.

**Comment by delta003** [comment]
👍  I was hoping to get https://github.com/palantir/python-jsonrpc-server/pull/40 in next release so waiting a day for response

File: setup.py [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
+import sys
 
 README = open('README.rst', 'r').read()
 
+install_requires = [
+        'configparser; python_version<"3.0"',
+        'future>=0.14.0; python_version<"3"',
+        'backports.functools_lru_cache; python_version<"3.2"',
+        'jedi>=0.17.0,<0.18.0',
+        'python-jsonrpc-server>=0.3.2',
```
Comment (ccordoba12) [inline setup.py]:
```suggestion
        'python-jsonrpc-server>=0.4.0',
```

@steff456, with this change we'll bring compatibility between this project and `python-jsonrpc-server` again.

**Review by ccordoba12 — COMMENTED** [review]
@steff456, one minor comment for you, then this should be ready.

**Pull request merged**